### ISSUE TITLE
2.13.1: test suite is failing

### ISSUE THREAD

**Issue by kloczek** [akheron/jansson#574]
I'm using binutils from fedora rawhide (2.35.1-34.fc34) and jansson library is optimised usimg LTO.
Looks lie `nm` output chas hanged a bit and that is confusing `api/check-exports` test unit.
```
=================================================================
api/check-exports
=================================================================
--- logs/api/check-exports/exports      2021-03-08 03:15:14.213977724 +0000
+++ logs/api/check-exports/output       2021-03-08 03:15:14.253977709 +0000
@@ -1,81 +1,81 @@
-jansson_version_cmp
-jansson_version_str
-json_array
-json_array_append_new
-json_array_clear
-json_array_extend
-json_array_get
-json_array_insert_new
-json_array_remove
-json_array_set_new
-json_array_size
-json_copy
-json_deep_copy
-json_delete
-json_dump_callback
-json_dump_file
-json_dumpb
-json_dumpf
-json_dumpfd
-json_dumps
-json_equal
-json_false
-json_get_alloc_funcs
-json_integer
-json_integer_set
-json_integer_value
-json_load_callback
-json_load_file
-json_loadb
-json_loadf
-json_loadfd
-json_loads
-json_null
-json_number_value
-json_object
-json_object_clear
-json_object_del
-json_object_deln
-json_object_get
-json_object_getn
-json_object_iter
-json_object_iter_at
-json_object_iter_key
-json_object_iter_key_len
-json_object_iter_next
-json_object_iter_set_new
-json_object_iter_value
-json_object_key_to_iter
-json_object_seed
-json_object_set_new
-json_object_set_new_nocheck
-json_object_setn_new
-json_object_setn_new_nocheck
-json_object_size
-json_object_update
-json_object_update_existing
-json_object_update_missing
-json_object_update_recursive
-json_pack
-json_pack_ex
-json_real
-json_real_set
-json_real_value
-json_set_alloc_funcs
-json_sprintf
-json_string
-json_string_length
-json_string_nocheck
-json_string_set
-json_string_set_nocheck
-json_string_setn
-json_string_setn_nocheck
-json_string_value
-json_stringn
-json_stringn_nocheck
-json_true
-json_unpack
-json_unpack_ex
-json_vpack_ex
-json_vsprintf
-json_vunpack_ex
+jansson_version_cmp@@libjansson.so.4
+jansson_version_str@@libjansson.so.4
+json_array@@libjansson.so.4
+json_array_append_new@@libjansson.so.4
+json_array_clear@@libjansson.so.4
+json_array_extend@@libjansson.so.4
+json_array_get@@libjansson.so.4
+json_array_insert_new@@libjansson.so.4
+json_array_remove@@libjansson.so.4
+json_array_set_new@@libjansson.so.4
+json_array_size@@libjansson.so.4
+json_copy@@libjansson.so.4
+json_deep_copy@@libjansson.so.4
+json_delete@@libjansson.so.4
+json_dump_callback@@libjansson.so.4
+json_dump_file@@libjansson.so.4
+json_dumpb@@libjansson.so.4
+json_dumpf@@libjansson.so.4
+json_dumpfd@@libjansson.so.4
+json_dumps@@libjansson.so.4
+json_equal@@libjansson.so.4
+json_false@@libjansson.so.4
+json_get_alloc_funcs@@libjansson.so.4
+json_integer@@libjansson.so.4
+json_integer_set@@libjansson.so.4
+json_integer_value@@libjansson.so.4
+json_load_callback@@libjansson.so.4
+json_load_file@@libjansson.so.4
+json_loadb@@libjansson.so.4
+json_loadf@@libjansson.so.4
+json_loadfd@@libjansson.so.4
+json_loads@@libjansson.so.4
+json_null@@libjansson.so.4
+json_number_value@@libjansson.so.4
+json_object@@libjansson.so.4
+json_object_clear@@libjansson.so.4
+json_object_del@@libjansson.so.4
+json_object_deln@@libjansson.so.4
+json_object_get@@libjansson.so.4
+json_object_getn@@libjansson.so.4
+json_object_iter@@libjansson.so.4
+json_object_iter_at@@libjansson.so.4
+json_object_iter_key@@libjansson.so.4
+json_object_iter_key_len@@libjansson.so.4
+json_object_iter_next@@libjansson.so.4
+json_object_iter_set_new@@libjansson.so.4
+json_object_iter_value@@libjansson.so.4
+json_object_key_to_iter@@libjansson.so.4
+json_object_seed@@libjansson.so.4
+json_object_set_new@@libjansson.so.4
+json_object_set_new_nocheck@@libjansson.so.4
+json_object_setn_new@@libjansson.so.4
+json_object_setn_new_nocheck@@libjansson.so.4
+json_object_size@@libjansson.so.4
+json_object_update@@libjansson.so.4
+json_object_update_existing@@libjansson.so.4
+json_object_update_missing@@libjansson.so.4
+json_object_update_recursive@@libjansson.so.4
+json_pack@@libjansson.so.4
+json_pack_ex@@libjansson.so.4
+json_real@@libjansson.so.4
+json_real_set@@libjansson.so.4
+json_real_value@@libjansson.so.4
+json_set_alloc_funcs@@libjansson.so.4
+json_sprintf@@libjansson.so.4
+json_string@@libjansson.so.4
+json_string_length@@libjansson.so.4
+json_string_nocheck@@libjansson.so.4
+json_string_set@@libjansson.so.4
+json_string_set_nocheck@@libjansson.so.4
+json_string_setn@@libjansson.so.4
+json_string_setn_nocheck@@libjansson.so.4
+json_string_value@@libjansson.so.4
+json_stringn@@libjansson.so.4
+json_stringn_nocheck@@libjansson.so.4
+json_true@@libjansson.so.4
+json_unpack@@libjansson.so.4
+json_unpack_ex@@libjansson.so.4
+json_vpack_ex@@libjansson.so.4
+json_vsprintf@@libjansson.so.4
+json_vunpack_ex@@libjansson.so.4

=================================================================
```
content of the test/logs/api/check-exports:
```
                 w _ITM_deregisterTMCloneTable
                 w _ITM_registerTMCloneTable
                 U __assert_fail@@GLIBC_2.2.5
                 w __cxa_finalize@@GLIBC_2.2.5
                 U __errno_location@@GLIBC_2.2.5
                 w __gmon_start__
                 U __snprintf_chk@@GLIBC_2.3.4
                 U __stack_chk_fail@@GLIBC_2.4
                 U __strncpy_chk@@GLIBC_2.3.4
                 U __vsnprintf_chk@@GLIBC_2.3.4
                 U close@@GLIBC_2.2.5
                 U fclose@@GLIBC_2.2.5
                 U fgetc@@GLIBC_2.2.5
                 U fopen@@GLIBC_2.2.5
                 U free@@GLIBC_2.2.5
                 U fwrite@@GLIBC_2.2.5
                 U getpid@@GLIBC_2.2.5
                 U gettimeofday@@GLIBC_2.2.5
000000000000aca0 T jansson_version_cmp@@libjansson.so.4
000000000000ac90 T jansson_version_str@@libjansson.so.4
00000000000050e0 T json_array@@libjansson.so.4
0000000000008860 T json_array_append_new@@libjansson.so.4
000000000000a7d0 T json_array_clear@@libjansson.so.4
00000000000051b0 T json_array_extend@@libjansson.so.4
0000000000005180 T json_array_get@@libjansson.so.4
000000000000a590 T json_array_insert_new@@libjansson.so.4
000000000000a700 T json_array_remove@@libjansson.so.4
00000000000087b0 T json_array_set_new@@libjansson.so.4
0000000000005160 T json_array_size@@libjansson.so.4
000000000000aa20 T json_copy@@libjansson.so.4
000000000000abb0 T json_deep_copy@@libjansson.so.4
0000000000005ad0 T json_delete@@libjansson.so.4
0000000000007d20 T json_dump_callback@@libjansson.so.4
0000000000007f90 T json_dump_file@@libjansson.so.4
0000000000007f10 T json_dumpb@@libjansson.so.4
0000000000007f70 T json_dumpf@@libjansson.so.4
0000000000007ff0 T json_dumpfd@@libjansson.so.4
0000000000007e30 T json_dumps@@libjansson.so.4
000000000000a850 T json_equal@@libjansson.so.4
00000000000059c0 T json_false@@libjansson.so.4
0000000000004e20 T json_get_alloc_funcs@@libjansson.so.4
0000000000005810 T json_integer@@libjansson.so.4
0000000000005860 T json_integer_set@@libjansson.so.4
0000000000005840 T json_integer_value@@libjansson.so.4
0000000000009890 T json_load_callback@@libjansson.so.4
00000000000095f0 T json_load_file@@libjansson.so.4
00000000000092d0 T json_loadb@@libjansson.so.4
0000000000009470 T json_loadf@@libjansson.so.4
00000000000096d0 T json_loadfd@@libjansson.so.4
0000000000009150 T json_loads@@libjansson.so.4
00000000000059d0 T json_null@@libjansson.so.4
0000000000005960 T json_number_value@@libjansson.so.4
0000000000004e50 T json_object@@libjansson.so.4
0000000000005c50 T json_object_clear@@libjansson.so.4
0000000000005e60 T json_object_del@@libjansson.so.4
0000000000005e30 T json_object_deln@@libjansson.so.4
0000000000004f30 T json_object_get@@libjansson.so.4
0000000000004f00 T json_object_getn@@libjansson.so.4
0000000000004f70 T json_object_iter@@libjansson.so.4
0000000000004fa0 T json_object_iter_at@@libjansson.so.4
0000000000005070 T json_object_iter_key@@libjansson.so.4
0000000000005090 T json_object_iter_key_len@@libjansson.so.4
0000000000005030 T json_object_iter_next@@libjansson.so.4
0000000000008710 T json_object_iter_set_new@@libjansson.so.4
00000000000050a0 T json_object_iter_value@@libjansson.so.4
00000000000050c0 T json_object_key_to_iter@@libjansson.so.4
0000000000004ce0 T json_object_seed@@libjansson.so.4
00000000000086a0 T json_object_set_new@@libjansson.so.4
00000000000083f0 T json_object_set_new_nocheck@@libjansson.so.4
0000000000008610 T json_object_setn_new@@libjansson.so.4
0000000000008020 T json_object_setn_new_nocheck@@libjansson.so.4
0000000000004ed0 T json_object_size@@libjansson.so.4
0000000000008460 T json_object_update@@libjansson.so.4
0000000000008210 T json_object_update_existing@@libjansson.so.4
0000000000008520 T json_object_update_missing@@libjansson.so.4
0000000000008300 T json_object_update_recursive@@libjansson.so.4
000000000000a4c0 T json_pack@@libjansson.so.4
000000000000a410 T json_pack_ex@@libjansson.so.4
0000000000005880 T json_real@@libjansson.so.4
0000000000005920 T json_real_set@@libjansson.so.4
0000000000005900 T json_real_value@@libjansson.so.4
0000000000004e00 T json_set_alloc_funcs@@libjansson.so.4
0000000000005750 T json_sprintf@@libjansson.so.4
0000000000005460 T json_string@@libjansson.so.4
00000000000054b0 T json_string_length@@libjansson.so.4
00000000000052f0 T json_string_nocheck@@libjansson.so.4
00000000000055f0 T json_string_set@@libjansson.so.4
0000000000005560 T json_string_set_nocheck@@libjansson.so.4
00000000000055a0 T json_string_setn@@libjansson.so.4
00000000000054d0 T json_string_setn_nocheck@@libjansson.so.4
0000000000005490 T json_string_value@@libjansson.so.4
0000000000005410 T json_stringn@@libjansson.so.4
0000000000005380 T json_stringn_nocheck@@libjansson.so.4
00000000000059b0 T json_true@@libjansson.so.4
00000000000070d0 T json_unpack@@libjansson.so.4
0000000000007020 T json_unpack_ex@@libjansson.so.4
000000000000a250 T json_vpack_ex@@libjansson.so.4
0000000000005630 T json_vsprintf@@libjansson.so.4
0000000000006e20 T json_vunpack_ex@@libjansson.so.4
0000000000000000 A libjansson.so.4
                 U localeconv@@GLIBC_2.2.5
                 U malloc@@GLIBC_2.2.5
                 U memchr@@GLIBC_2.2.5
                 U memcmp@@GLIBC_2.2.5
                 U memcpy@@GLIBC_2.14
                 U memmove@@GLIBC_2.2.5
                 U open@@GLIBC_2.2.5
                 U qsort@@GLIBC_2.2.5
                 U read@@GLIBC_2.2.5
                 U sched_yield@@GLIBC_2.2.5
                 U stdin@@GLIBC_2.2.5
                 U strchr@@GLIBC_2.2.5
                 U strcmp@@GLIBC_2.2.5
                 U strerror@@GLIBC_2.2.5
                 U strlen@@GLIBC_2.2.5
                 U strncpy@@GLIBC_2.2.5
                 U strtod@@GLIBC_2.2.5
                 U strtoll@@GLIBC_2.2.5
                 U write@@GLIBC_2.2.5
```

**Comment by kloczek** [comment]
Just checked 2.14 and looks like issue still is aroud ..
```console
/usr/bin/make  check-TESTS
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14/test'
FAIL: run-suites
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14/test'
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14/test'
=======================================
   jansson 2.14: test/test-suite.log
=======================================

# TOTAL: 1
# PASS:  0
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: run-suites
================

Suite: api
check-exports... FAILED
test_array.c... ok
test_chaos.c... ok
test_copy.c... ok
test_dump.c... ok
test_dump_callback.c... ok
test_equal.c... ok
test_fixed_size.c... ok
test_load.c... ok
test_load_callback.c... ok
test_loadb.c... ok
test_memory_funcs.c... ok
test_number.c... ok
test_object.c... ok
test_pack.c... ok
test_simple.c... ok
test_sprintf.c... ok
test_unpack.c... ok
test_version.c... ok
=================================================================
api/check-exports
=================================================================
--- logs/api/check-exports/exports      2021-09-09 19:27:40.966854304 +0100
+++ logs/api/check-exports/output       2021-09-09 19:27:41.020854480 +0100
@@ -1,81 +1,81 @@
-jansson_version_cmp
-jansson_version_str
-json_array
-json_array_append_new
-json_array_clear
-json_array_extend
-json_array_get
-json_array_insert_new
-json_array_remove
-json_array_set_new
-json_array_size
-json_copy
-json_deep_copy
-json_delete
-json_dump_callback
-json_dump_file
-json_dumpb
-json_dumpf
-json_dumpfd
-json_dumps
-json_equal
-json_false
-json_get_alloc_funcs
-json_integer
-json_integer_set
-json_integer_value
-json_load_callback
-json_load_file
-json_loadb
-json_loadf
-json_loadfd
-json_loads
-json_null
-json_number_value
-json_object
-json_object_clear
-json_object_del
-json_object_deln
-json_object_get
-json_object_getn
-json_object_iter
-json_object_iter_at
-json_object_iter_key
-json_object_iter_key_len
-json_object_iter_next
-json_object_iter_set_new
-json_object_iter_value
-json_object_key_to_iter
-json_object_seed
-json_object_set_new
-json_object_set_new_nocheck
-json_object_setn_new
-json_object_setn_new_nocheck
-json_object_size
-json_object_update
-json_object_update_existing
-json_object_update_missing
-json_object_update_recursive
-json_pack
-json_pack_ex
-json_real
-json_real_set
-json_real_value
-json_set_alloc_funcs
-json_sprintf
-json_string
-json_string_length
-json_string_nocheck
-json_string_set
-json_string_set_nocheck
-json_string_setn
-json_string_setn_nocheck
-json_string_value
-json_stringn
-json_stringn_nocheck
-json_true
-json_unpack
-json_unpack_ex
-json_vpack_ex
-json_vsprintf
-json_vunpack_ex
+jansson_version_cmp@@libjansson.so.4
+jansson_version_str@@libjansson.so.4
+json_array@@libjansson.so.4
+json_array_append_new@@libjansson.so.4
+json_array_clear@@libjansson.so.4
+json_array_extend@@libjansson.so.4
+json_array_get@@libjansson.so.4
+json_array_insert_new@@libjansson.so.4
+json_array_remove@@libjansson.so.4
+json_array_set_new@@libjansson.so.4
+json_array_size@@libjansson.so.4
+json_copy@@libjansson.so.4
+json_deep_copy@@libjansson.so.4
+json_delete@@libjansson.so.4
+json_dump_callback@@libjansson.so.4
+json_dump_file@@libjansson.so.4
+json_dumpb@@libjansson.so.4
+json_dumpf@@libjansson.so.4
+json_dumpfd@@libjansson.so.4
+json_dumps@@libjansson.so.4
+json_equal@@libjansson.so.4
+json_false@@libjansson.so.4
+json_get_alloc_funcs@@libjansson.so.4
+json_integer@@libjansson.so.4
+json_integer_set@@libjansson.so.4
+json_integer_value@@libjansson.so.4
+json_load_callback@@libjansson.so.4
+json_load_file@@libjansson.so.4
+json_loadb@@libjansson.so.4
+json_loadf@@libjansson.so.4
+json_loadfd@@libjansson.so.4
+json_loads@@libjansson.so.4
+json_null@@libjansson.so.4
+json_number_value@@libjansson.so.4
+json_object@@libjansson.so.4
+json_object_clear@@libjansson.so.4
+json_object_del@@libjansson.so.4
+json_object_deln@@libjansson.so.4
+json_object_get@@libjansson.so.4
+json_object_getn@@libjansson.so.4
+json_object_iter@@libjansson.so.4
+json_object_iter_at@@libjansson.so.4
+json_object_iter_key@@libjansson.so.4
+json_object_iter_key_len@@libjansson.so.4
+json_object_iter_next@@libjansson.so.4
+json_object_iter_set_new@@libjansson.so.4
+json_object_iter_value@@libjansson.so.4
+json_object_key_to_iter@@libjansson.so.4
+json_object_seed@@libjansson.so.4
+json_object_set_new@@libjansson.so.4
+json_object_set_new_nocheck@@libjansson.so.4
+json_object_setn_new@@libjansson.so.4
+json_object_setn_new_nocheck@@libjansson.so.4
+json_object_size@@libjansson.so.4
+json_object_update@@libjansson.so.4
+json_object_update_existing@@libjansson.so.4
+json_object_update_missing@@libjansson.so.4
+json_object_update_recursive@@libjansson.so.4
+json_pack@@libjansson.so.4
+json_pack_ex@@libjansson.so.4
+json_real@@libjansson.so.4
+json_real_set@@libjansson.so.4
+json_real_value@@libjansson.so.4
+json_set_alloc_funcs@@libjansson.so.4
+json_sprintf@@libjansson.so.4
+json_string@@libjansson.so.4
+json_string_length@@libjansson.so.4
+json_string_nocheck@@libjansson.so.4
+json_string_set@@libjansson.so.4
+json_string_set_nocheck@@libjansson.so.4
+json_string_setn@@libjansson.so.4
+json_string_setn_nocheck@@libjansson.so.4
+json_string_value@@libjansson.so.4
+json_stringn@@libjansson.so.4
+json_stringn_nocheck@@libjansson.so.4
+json_true@@libjansson.so.4
+json_unpack@@libjansson.so.4
+json_unpack_ex@@libjansson.so.4
+json_vpack_ex@@libjansson.so.4
+json_vsprintf@@libjansson.so.4
+json_vunpack_ex@@libjansson.so.4

=================================================================
Suite: encoding-flags
array... ok
compact-array... ok
compact-object... ok
ensure-ascii... ok
indent-array... ok
indent-compact-array... ok
indent-compact-object... ok
indent-object... ok
object... ok
preserve-order... ok
real-precision... ok
sort-keys... ok
Suite: invalid
apostrophe... ok
ascii-unicode-identifier... ok
brace-comma... ok
bracket-comma... ok
bracket-one-comma... ok
empty... ok
extra-comma-in-array... ok
extra-comma-in-multiline-array... ok
garbage-after-newline... ok
garbage-at-the-end... ok
integer-starting-with-zero... ok
invalid-escape... ok
invalid-identifier... ok
invalid-negative-integer... ok
invalid-negative-real... ok
invalid-second-surrogate... ok
invalid-unicode-escape... ok
lone-open-brace... ok
lone-open-bracket... ok
lone-second-surrogate... ok
minus-sign-without-number... ok
negative-integer-starting-with-zero... ok
null... ok
null-byte-in-object-key... ok
null-byte-in-string... ok
null-byte-outside-string... ok
null-escape-in-string... ok
object-apostrophes... ok
object-garbage-at-end... ok
object-in-unterminated-array... ok
object-no-colon... ok
object-no-value... ok
object-unterminated-value... ok
real-garbage-after-e... ok
real-negative-overflow... ok
real-positive-overflow... ok
real-truncated-at-e... ok
real-truncated-at-point... ok
recursion-depth... ok
tab-character-in-string... ok
too-big-negative-integer... ok
too-big-positive-integer... ok
truncated-unicode-surrogate... ok
unicode-identifier... ok
unterminated-array... ok
unterminated-array-and-object... ok
unterminated-empty-key... ok
unterminated-key... ok
unterminated-object-and-array... ok
unterminated-string... ok
Suite: invalid-unicode
encoded-surrogate-half... ok
invalid-utf-8-after-backslash... ok
invalid-utf-8-in-array... ok
invalid-utf-8-in-bigger-int... ok
invalid-utf-8-in-escape... ok
invalid-utf-8-in-exponent... ok
invalid-utf-8-in-identifier... ok
invalid-utf-8-in-int... ok
invalid-utf-8-in-real-after-e... ok
invalid-utf-8-in-string... ok
lone-invalid-utf-8... ok
lone-utf-8-continuation-byte... ok
not-in-unicode-range... ok
overlong-3-byte-encoding... ok
overlong-4-byte-encoding... ok
overlong-ascii-encoding... ok
restricted-utf-8... ok
truncated-utf-8... ok
Suite: valid
complex-array... ok
empty-array... ok
empty-object... ok
empty-object-in-array... ok
empty-string... ok
escaped-utf-control-char... ok
false... ok
negative-int... ok
negative-one... ok
negative-zero... ok
null... ok
one-byte-utf-8... ok
real-capital-e... ok
real-capital-e-negative-exponent... ok
real-capital-e-positive-exponent... ok
real-exponent... ok
real-fraction-exponent... ok
real-negative-exponent... ok
real-positive-exponent... ok
real-subnormal-number... ok
real-underflow... ok
short-string... ok
simple-ascii-string... ok
simple-int-0... ok
simple-int-1... ok
simple-int-123... ok
simple-object... ok
simple-real... ok
string-escapes... ok
three-byte-utf-8... ok
true... ok
two-byte-utf-8... ok
utf-8-string... ok
utf-surrogate-four-byte-encoding... ok
1 of 5 test suites failed
FAIL run-suites (exit status: 1)

============================================================================
Testsuite summary for jansson 2.14
============================================================================
# TOTAL: 1
# PASS:  0
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See test/test-suite.log
Please report to https://github.com/akheron/jansson/issues
============================================================================
```

**Comment by akheron** [comment]
I also get this locally. For some reason the tests are passing on CI :shrug:

**Comment by akheron** [comment]
Fix in #593

**Comment by kloczek** [comment]
yep just tested that PR and looks like it fixes tst suite:
```console
make[3]: Nothing to be done for 'check-am'.
Making check in ossfuzz
make[2]: Nothing to be done for 'check'.
/usr/bin/make  check-TESTS
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14/test'
PASS: run-suites
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14/test'
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14/test'
============================================================================
Testsuite summary for jansson 2.14
============================================================================
# TOTAL: 1
# PASS:  1
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14/test'
/usr/bin/make  check-TESTS
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14'
PASS: scripts/clang-format-check
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14'
============================================================================
Testsuite summary for jansson 2.14
============================================================================
# TOTAL: 1
# PASS:  1
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/jansson-2.14'
```
Thank you :)
